recipes/telegram/index.js

```diff
@@ -1 +1,21 @@
-module.exports = Ferdium => Ferdium;
+function instrumenEnvironment(webview) {
+  webview.executeJavaScript(`
+    (function() {
+        if(window.electron) {
+            return;
+         }
+        
+        window.electron = { };
+    })();
+  `);
+}
+module.exports = (Franz) =>
+  class Telegram extends Franz {
```

Comment:
Can we use `Ferdium` and extend `Ferdium` please?

Reply:
Updated in https://github.com/ferdium/ferdium-recipes/pull/439/commits/172252a27ba8b60aaabb4d126b82ad719bfcd8f8